content/guides/embed/ui-elements/explorer.md

```diff
@@ -264,7 +271,175 @@ more, see [Dedicated Scopes for Box UI Elements][scopes].
 
+## Metadata view
+
+With Content Explorer you can also display files
+and folders based on their metadata.
+This view is called the metadata view and uses
+metadata template and metadata query to find the data you
+want to display.
+
+### Prerequisites
+
+- Node version 18.18.2 or higher
+- React version 17.0.2 or higher (React 18 is not supported)
+- BUIE version 19.0.0 or higher
+
+### Create and configure an app
+
+  1. [Create a Box app][box-app].
+  2. Add the local development address in the CORS Domains:
+    ![CORS Domains](./images/box-app-cors.png)
+  3. Generate a [developer token][token].
+
+### Create a metadata template
+
+The next step is to create a metadata template you will use to populate the Content Explorer.
+
+1. Create a metadata template. You can use [Metadata API][creating-templates-api] or [Admin Console][creating-templates-ui] to do so.
+2. Apply an already created template to a Box folder. Make sure you enable the cascade policy. For detailed instructions, see [instructions on customizing and applying templates][apply-templates].
+
+### Display Metadata View
+
+To make things easier, you can use a [sample project][metadata-project] to launch metadata view.
+
+1. Clone the metadata sample project.
+2. Update the placeholders in [`App.js`][appjs] with actual values:
+
+      | Parameter          |  Description |
+      |--------------------|-----------------|
+      |`DEVELOPER_TOKEN`   |[Developer token][token] generated in the the Developer Console. |
+      |`ENTERPRISE_ID`     |Enterprise ID copied from the **General Settings** tab of your application.|
+      |`METADATA_TEMPLATE_NAME`  |Name of your already created metadata template. The name must be lowercase, for example `mymetadatatemplate`.|
+      |`ROOTFOLDER_ID`   |  ID of Box folder to which you applied the metadata template. |
+
+The `defaultView`, `fieldsToShow`, and `metadataQuery` parameters are already defined in the sample project, as in the example below.
+
+For additional information on metadata queries, see [this guide][metadata-query]
+
+3. Pass the required parameters to the Content Explorer component.
+
+```js
+[...]
+
+function App() {
+  [...]
+
+  return (
+      <IntlProvider locale="en">
+        <div className="App">
+          <header className="App-header">
+            <h2>Metadata view in Content Explorer</h2>
+          </header>
+          <section>
+            <div className="metadata-based-view">
+              <ContentExplorer
+                rootFolderId={rootFolderID}
+                token={token}
+                metadataQuery={metadataQuery}
+                fieldsToShow={fieldsToShow}
+                defaultView={defaultView}
+              />
+            </div>
+          </section>
+        </div>
+      </IntlProvider>
+  );
+}
+
+export default App;
+```
+
+A sample React component including metadata will look similar to the following:
```

Comment:
Maybe something like this? 
A sample code for a React component including the Content Explorer metadata view would look as following.